docs/release.md

```diff
@@ -39,33 +39,81 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, or about every 15 weeks beginning
+on the first Monday in January.  This is the same [release cycle](https://kubernetes.io/releases/release/) as Kubernetes, but KubeVirt will trail Kubernetes releases
+by 1 to 4 weeks.  Here's an example release cycle using the 2022 calendar:
```

Comment:
Nit: Can we separate the example from the statement:

```
by 1 to 4 weeks.

Here's an example …
```

Mainly to make the statement really clear.